documentation/error-handling.md

```diff
@@ -0,0 +1,113 @@
+# Error handling
```

Comment:
We should maybe add a quick note/ links for connecting react error boundaries to these services for developers writing react extensions

Reply:
Are you thinking about using ErrorBoundary for React? I don't think it can catch everything, like importScripts... I don't think there is anything specific to React that we could highlight here.

Reply:
I was just thinking of a one-sentence thing, something like "If you are writing your UI extension in React, you can follow Bugsnag’s [React integration guide](https://docs.bugsnag.com/platforms/javascript/legacy/react/) to get additional context on errors thrown while rendering your components."